DataManagementSystem/Utilities/MetaQuery.py

```diff
@@ -0,0 +1,208 @@
+########################################################################
```

Comment:
Hi Andrei,
I did not realize that there was already a MetaQuery class in DataManagementSystem/Client, so I took this old version that I had in my own fork. 
I think it would be better to use the one under  DataManagementSystem/Client and remove that in DataManagementSystem/Utilities. I've just tried all tests and they work fine.